recipe/meta.yaml

```
@@ -44,27 +37,34 @@ requirements:
     - py_stringsimjoin >=0.3.0
     - cloudpickle >=0.2.1
     - pyparsing >=2.1.4
-    - scikit-learn >=0.18,<0.22.0a0
-    - xgboost  # [not win]
-    - pyqt
+    - scikit-learn >=0.22
     - requests
-    - ipython  # ==5.6
+    - ipython >=5.6
     - matplotlib-base >=2.2.4
```

```suggestion
    - matplotlib-base >=2.2.4
    - scipy
```
https://inspector.pypi.io/project/py-entitymatching/0.4.2/packages/3d/77/81c61707e98558cadac41f7e7aaf15f217cdd5cc6f106fdec57d8b36eff3/py-entitymatching-0.4.2.tar.gz/py-entitymatching-0.4.2/setup.py#line.125

---

Hmm, I overlooked that -- but the tests don't fail.
(Maybe they don't test the `scipy` parts...)